content/chainguard/chainguard-images/chainguard-registry/pull-through-guides/artifactory/artifactory-packages-pull-through/index.md

```diff
@@ -217,43 +296,42 @@ export VIRTUAL_TOKEN=my-virtual-repo-token
 
 You'll use this token to create one more Dockerfile in the next section.
 
-### Testing pull through with a virtual package repository
+### Testing pull-through with a virtual package repository
```

Comment:
Honestly .. i would even collapse this more now that I see .. create all three repos and the virtual repo in one section

Then have one section that shows a docker build that accesses the group in a build

Reply:
I'm going to hold off on this for now. Partly because I like the current structure and partly because I want to get this off my plate. But I do think what you suggest might be the ideal.